### PR TITLE
Cut release for node 8.6

### PR DESCRIPTION
**Pull request by teodanciu**
# Description

8.6 release: 
* ouroboros-consensus-0.13.0.0
* ouroboros-consensus-cardano-0.11.0.0
* ouroboros-consensus-diffusion-0.8.0.2
* ouroboros-consensus-protocol-0.6.0.0

### CODE REVIEW

File: ouroboros-consensus-diffusion/ouroboros-consensus-diffusion.cabal [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cabal-version:   3.0
 name:            ouroboros-consensus-diffusion
-version:         0.8.0.1
+version:         0.9.0.0
```
Comment (jasagredo) [inline ouroboros-consensus-diffusion/ouroboros-consensus-diffusion.cabal:3]: This was just a patch change. Only the patch version should increase, i.e. 0.8.0.2

**Comment by jasagredo** [comment]
Also you need to update the badges in the README.md file.

File: ouroboros-consensus-diffusion/ouroboros-consensus-diffusion.cabal (hunk 1 above)
Comment (teodanciu) [reply ouroboros-consensus-diffusion/ouroboros-consensus-diffusion.cabal]: Fixed.

**Comment by teodanciu** [comment]
Force pushed, because I edited the commit created by the release script, to correct the version (0.8.0.2), to avoid confusion.

**Review by amesgen — APPROVED** [review]

**Pull request merged**